recipes-multimedia/gstreamer/gstreamer1.0-plugins-base/0001-glupload-Add-VideoMetas-and-GLSyncMeta-to-the-raw-up.patch

```diff
@@ -0,0 +1,86 @@
+From 8d32de090554cf29fe359f83aa46000ba658a693 Mon Sep 17 00:00:00 2001
+From: Thibault Saunier <tsaunier@igalia.com>
+Date: Fri, 27 Sep 2019 11:10:43 -0300
+Subject: [PATCH] glupload: Add VideoMetas and GLSyncMeta to the raw uploaded
+ buffers
+
+This is done by reusing `gst_gl_memory_setup_buffer` avoiding to
+duplicate code.
+
+Without a VideoMeta, mapping those buffers lead to GstBuffer mapping the
+buffer in system memory even when specifying the GL flags (through the
+buffer merging mechanism) making the result totally broken.
+
+Upstream-Status: Backport
```

Comment:
Can you please put a link to or SHA of the original commit? Would be easier to track when `1.16.2` is integrated.

I believe the link is the one you put in the commit message (https://gitlab.freedesktop.org/gstreamer/gst-plugins-base/-/merge_requests/444), would be useful to have it here as well.

Reply:
Done.